src/object-properties.js

```diff
@@ -147,13 +147,15 @@ const properties = {
  * @property {boolean=} hoverEffect - Toggles hover effect
  * @property {PaletteColor=} hoverColor - Background hover color. Uses `#f4f4f4` if no hover colors are set, is transparent if only `hoverFontColor` is set
  * @property {PaletteColor=} hoverFontColor - When only `hoverColor` is set, this is adjusted to either `#f4f4f4` or `#ffffff` for optimal contrast
+ * @property {string=} padding - padding for the body cells on css format
```

Comment:
This is available if you write the props yourself, but wont exist in styling panel later. I guess we be documented